### PR TITLE
Accessible Javalab dropdowns

### PR DESCRIPTION
**Pull request by bencodeorg**
TL;DR of this is that you can now tab navigate (and execute onclicks via enter key) to dropdown menus in javalab (ie, settings, file list, and file options). This involved a couple of changes:

1. Move all `a` tags in these dropdowns to `button` tags -- we don't do any navigation to other pages via these dropdowns, and the accessibility standard is to use button tags in those cases. It also gives us things like tab navigation and enter presses execute onClick for for free. Some styling updates required to get this to work.
2. Move file option dropdown (`JavalabEditorTabMenu`) before editor window (and after tabs), which makes sense structurally and allows tab navigation into these options.

**After these changes (all navigation in video via keyboard)**

https://user-images.githubusercontent.com/25372625/167450073-b8d1e96a-e513-4149-bf8c-9346fc04e83c.mov

## Links

- [JAVA-564](https://codedotorg.atlassian.net/browse/JAVA-564)

## Testing story

Tested manually that I could keyboard navigate to each of the dropdown menus affected by this change.

## Follow-up work

We'll need to make it more obvious that you can tab navigate out of the editor by hitting the escape key, then hitting tab (or shift+tab). That work is in [this Jira task](https://codedotorg.atlassian.net/browse/JAVA-573).

### CODE REVIEW

**Review by molly-moen — APPROVED** [review]
🎉

**Review by sanchitmalhotra126 — APPROVED** [review]
Nice!

File: apps/src/javalab/JavalabEditorTabMenu.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import JavalabDropdown from './components/JavalabDropdown';
 import javalabMsg from '@cdo/javalab/locale';
```
Comment (jmkulwik) [inline apps/src/javalab/JavalabEditorTabMenu.jsx:4]: If you get the chance, this would be a good opportunity to also add reactOnClickOutside so the dropdown auto-closes when clicking out of it (similar to JavaLabFileExplorer)

**Review by jmkulwik — APPROVED** [review]
Really good work here.

File: apps/src/javalab/JavalabEditorTabMenu.jsx (hunk 1 above)
Comment (bencodeorg) [reply apps/src/javalab/JavalabEditorTabMenu.jsx:4]: I think @sanchitmalhotra126 did that during the hackathon, just needs to be merged!

https://github.com/code-dot-org/code-dot-org/pull/46025
